Scripts/fetch-prebuilt-wda.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const request = require('request-promise');
+const { asyncify } = require('asyncbox');
+const { logger, fs, mkdirp } = require('appium-support');
+const { exec } = require('teen_process');
+
+const log = logger.getLogger('WDA');
+
+async function fetchPrebuiltWebDriverAgentAssets () {
+  const tag = require('../package.json').version;
+  const downloadUrl = `https://api.github.com/repos/appium/webdriveragent/releases/tags/v${tag}`;
+  log.info(`Getting WDA release ${downloadUrl}`);
+  const releases = await request.get(downloadUrl, {
+    headers: {
+      'user-agent': 'node.js',
+    },
+    json: true,
+  });
+  const webdriveragentsDir = path.resolve(__dirname, '..', 'webdriveragents');
+  log.info(`Downloading assets to: ${webdriveragentsDir}`);
+  await fs.rimraf(webdriveragentsDir);
+  await mkdirp(webdriveragentsDir);
+  for (const asset of releases.assets) {
```

Comment:
we could download the stuff in parallel to make the operation faster, like in https://github.com/appium/appium-chromedriver/blob/8a6c414e6ae33600e5f4e5a6531c7ffd78072c4a/lib/storage-client.js#L204